integration_tests/test_pull_properties.py

```diff
@@ -68,7 +106,52 @@ def test_pull(self):
         self.assertTrue(len(state.assets['stage-packages']) > 0)
         self.assertTrue(len(state.assets['build-packages']) > 0)
         self.assertIn('hello=2.10-1', state.assets['stage-packages'])
-        self.assertIn('hello=2.10-1', state.assets['build-packages'])
+        self.assertIn('source-details', state.assets)
+
+    def test_pull_git(self):
```

Comment:
updated.  I'm thinking test_pull_properties.py isn't the best place for these tests.  Any objections to having all these in test_source_tracking.py?